src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -91,6 +91,10 @@
     /** The String "(null)". */
     private static final String NULL_STRING = "(null)";
 
+    /** Inlined config is not supported for non java files. */
+    private static final Set<String> PERMANENT_SUPPRESSED_CHECKS = new HashSet<>(Arrays.asList(
```

Comment:
it should be used in same place as SUPPRESSED_CHECKS as addditional list of suppression.